lib/report_formatter/timeline.rb

```diff
@@ -52,6 +52,13 @@ def to_cid(id)
       ApplicationRecord.compress_id(id)
     end
 
+    def tl_href(col, rec, text, ems_cloud, ems_container)
+      tl_message = TimelineMessage.new(rec, text, ems_cloud, ems_container)
```

Comment:
> @moolitayer will it be better if tl_message will be global, so not to initialize the class each call ?

I not sure how you can do this, since the parameters change on each call?